### PR TITLE
Created modules for a load balancer and DNS

### PR DESCRIPTION
**Pull request by khoovirajsingh**

### CODE REVIEW

File: modules/azure/dns/main.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+resource "azurerm_resource_group" "dns_rg" {
+  name     = var.resource_group_name
+  location = var.location
+}
+
+resource "azurerm_dns_zone" "dns_zone" {
```
Comment (gargath) [inline modules/azure/dns/main.tf:6]: We should not manage the zone itself as part of this module.
Instead look it up using a `data` block.

File: modules/azure/dns/variables.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+variable "dns_a_record_name" {
+  type        = string
+  description = "The host name"
+  default = "test"
+}
+
+variable "list_of_load_balancer_ips" {
+  type = list(string)
+  description = "The list of ips for the load balancers"
+  default = ["51.141.42.249", "20.49.204.139"]
```
Comment (gargath) [inline modules/azure/dns/variables.tf:28]: I don't think there should be a default for this - definitely not some random public IPs?

**Review by gargath — CHANGES_REQUESTED** [review]
Couple of comments here. Could you take another look?

File: modules/azure/dns/variables.tf (hunk 2 above)
Comment (khoovirajsingh) [reply modules/azure/dns/variables.tf]: yep good point

**Pull request merged**